.github/workflows/buildAndTestRyzenAI.yml

```diff
@@ -169,3 +169,39 @@ jobs:
           ninja check-programming-guide
 
           popd
+
+  build-quick-setup:
+    name: Run Examples on Ryzen AI using quick_setup.sh Environment
+    runs-on: amd7940hs
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: "true"
+
+      # Launch an ssh session via a proxy server if there is a need
+      # for debug. This seems to live for 35 min max
+      # https://github.com/mxschmitt/action-tmate
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
+        # To run this, launch it manually on the default branch and
+        # click on "launch_tmate_terminal_for_debug"
+        if: github.event_name == 'workflow_dispatch'
+                && inputs.launch_tmate_terminal_for_debug
+
+      - name: Run commands
+        run: |
+
+          pip cache purge
+          source /opt/xilinx/xrt/setup.sh
+          python -m pip install virtualenv
```

Comment:
Other workflows use venv. It is already installed, is part of python, and does not pip install anything to the gihub user's site packages.

Reply:
Makes sense to me! I've changed quick_setup to use venv instead of virtualenv.

Reply:
> Makes sense to me! I've changed quick_setup to use venv instead of virtualenv.

Note it was slightly religious statement I made (venv over virtualenv). There may be cases where virtualenv is less friction because ubuntu doesn't install venv by default and user might not have sudo to apt install it.

Reply:
I don't have a strong preference either way (venv vs virtualenv), but I think it makes sense to be consistent. Is there anyone else who may want to weigh in?